src/components/ResourceInstance/CreateResourceInstanceModel/CreateResourceInstanceModel.tsx

```
@@ -0,0 +1,193 @@
+              </ListItemIcon>
+
+              <>
+                <Box display={"flex"} flexDirection={"column"} gap={"10px"}>
+                  <BodyText>
+                    Your instance is being set up and will be ready shortly
+                    (usually within a few minutes). You can track its status in
+                    the dashboard. Below is the Instance ID for your reference.
+                  </BodyText>
+                  <InstanceIdContainer instanceId={instanceId} />
+                </Box>
+              </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)